<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="9" passed="6" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-11T13:49:01 IST" name="Manipal Capstone Project XYZ Bank Automation" finished-at="2024-03-11T13:50:13 IST" duration-ms="71324">
    <groups>
    </groups>
    <test started-at="2024-03-11T13:49:01 IST" name="Maniap Capstone Project Running on Chrome" finished-at="2024-03-11T13:50:13 IST" duration-ms="71324">
      <class name="com.mcp.qa.testcases.ExistingCustomerTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.mcp.qa.testcases.ExistingCustomerTest@18ce0030]" started-at="2024-03-11T13:49:36 IST" name="setUp" finished-at="2024-03-11T13:49:38 IST" duration-ms="2325" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="existingtest()[pri:0, instance:com.mcp.qa.testcases.ExistingCustomerTest@18ce0030]" started-at="2024-03-11T13:49:38 IST" name="existingtest" finished-at="2024-03-11T13:49:43 IST" duration-ms="5313" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- existingtest -->
      </class> <!-- com.mcp.qa.testcases.ExistingCustomerTest -->
      <class name="com.mcp.qa.testcases.TransactionDetailsTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.mcp.qa.testcases.TransactionDetailsTest@25d250c6]" started-at="2024-03-11T13:49:59 IST" name="setUp" finished-at="2024-03-11T13:50:02 IST" duration-ms="3628" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testCustomerLoginFlow()[pri:0, instance:com.mcp.qa.testcases.TransactionDetailsTest@25d250c6]" started-at="2024-03-11T13:50:02 IST" name="testCustomerLoginFlow" finished-at="2024-03-11T13:50:13 IST" duration-ms="10405" status="FAIL">
          <exception class="org.openqa.selenium.NoAlertPresentException">
            <message>
              <![CDATA[no such alert
  (Session info: chrome=122.0.6261.112)
Build info: version: '4.15.0', revision: '1d14b5521b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [348e42e0cd2d6c3e01690685e2cea9ab, getAlertText {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 122.0.6261.112, chrome: {chromedriverVersion: 122.0.6261.57 (87957e464700..., userDataDir: C:\Users\rohit\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52559}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52559/devtoo..., se:cdpVersion: 122.0.6261.112, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 348e42e0cd2d6c3e01690685e2cea9ab]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoAlertPresentException: no such alert
  (Session info: chrome=122.0.6261.112)
Build info: version: '4.15.0', revision: '1d14b5521b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [348e42e0cd2d6c3e01690685e2cea9ab, getAlertText {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 122.0.6261.112, chrome: {chromedriverVersion: 122.0.6261.57 (87957e464700..., userDataDir: C:\Users\rohit\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52559}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52559/devtoo..., se:cdpVersion: 122.0.6261.112, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 348e42e0cd2d6c3e01690685e2cea9ab
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:200)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:175)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:607)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:680)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:684)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.alert(RemoteWebDriver.java:1252)
at org.openqa.selenium.support.events.EventFiringWebDriver$EventFiringTargetLocator.alert(EventFiringWebDriver.java:767)
at com.mcp.qa.pages.CustomerDashboardPage.transactionsflow(CustomerDashboardPage.java:285)
at com.mcp.qa.testcases.TransactionDetailsTest.testCustomerLoginFlow(TransactionDetailsTest.java:25)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoAlertPresentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCustomerLoginFlow -->
      </class> <!-- com.mcp.qa.testcases.TransactionDetailsTest -->
      <class name="com.mcp.qa.testcases.DepositAndWithdrawalTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.mcp.qa.testcases.DepositAndWithdrawalTest@45b9a632]" started-at="2024-03-11T13:49:49 IST" name="setUp" finished-at="2024-03-11T13:49:51 IST" duration-ms="2323" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testCustomerLoginFlow()[pri:0, instance:com.mcp.qa.testcases.DepositAndWithdrawalTest@45b9a632]" started-at="2024-03-11T13:49:51 IST" name="testCustomerLoginFlow" finished-at="2024-03-11T13:49:59 IST" duration-ms="7772" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCustomerLoginFlow -->
      </class> <!-- com.mcp.qa.testcases.DepositAndWithdrawalTest -->
      <class name="com.mcp.qa.testcases.OpenAccountTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.mcp.qa.testcases.OpenAccountTest@309e345f]" started-at="2024-03-11T13:49:17 IST" name="setUp" finished-at="2024-03-11T13:49:21 IST" duration-ms="3156" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="managertest()[pri:1, instance:com.mcp.qa.testcases.OpenAccountTest@309e345f]" started-at="2024-03-11T13:49:21 IST" name="managertest" finished-at="2024-03-11T13:49:26 IST" duration-ms="5157" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- managertest -->
      </class> <!-- com.mcp.qa.testcases.OpenAccountTest -->
      <class name="com.mcp.qa.testcases.CustomerLoginAndDetailsTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.mcp.qa.testcases.CustomerLoginAndDetailsTest@4445629]" started-at="2024-03-11T13:49:43 IST" name="setUp" finished-at="2024-03-11T13:49:45 IST" duration-ms="2286" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testCustomerLoginFlow()[pri:0, instance:com.mcp.qa.testcases.CustomerLoginAndDetailsTest@4445629]" started-at="2024-03-11T13:49:45 IST" name="testCustomerLoginFlow" finished-at="2024-03-11T13:49:48 IST" duration-ms="3036" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCustomerLoginFlow -->
      </class> <!-- com.mcp.qa.testcases.CustomerLoginAndDetailsTest -->
      <class name="com.mcp.qa.testcases.AddCustomersTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.mcp.qa.testcases.AddCustomersTest@7a4ccb53]" started-at="2024-03-11T13:49:09 IST" name="setUp" finished-at="2024-03-11T13:49:11 IST" duration-ms="2200" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="customerDetails()[pri:1, instance:com.mcp.qa.testcases.AddCustomersTest@7a4ccb53]" started-at="2024-03-11T13:49:11 IST" name="customerDetails" finished-at="2024-03-11T13:49:13 IST" duration-ms="2191" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- customerDetails -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.mcp.qa.testcases.AddCustomersTest@7a4ccb53]" started-at="2024-03-11T13:49:13 IST" name="setUp" finished-at="2024-03-11T13:49:15 IST" duration-ms="2489" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="invalidCustomerDetails()[pri:2, instance:com.mcp.qa.testcases.AddCustomersTest@7a4ccb53]" started-at="2024-03-11T13:49:15 IST" name="invalidCustomerDetails" finished-at="2024-03-11T13:49:17 IST" duration-ms="1970" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Account is not added as the details are invalid expected [Please provide the valid Customer details] but found [Customer added successfully with customer id :6]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Account is not added as the details are invalid expected [Please provide the valid Customer details] but found [Customer added successfully with customer id :6]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at com.mcp.qa.pages.BankManagerDashboardPage.invalidCustomerDetails(BankManagerDashboardPage.java:247)
at com.mcp.qa.testcases.AddCustomersTest.invalidCustomerDetails(AddCustomersTest.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidCustomerDetails -->
      </class> <!-- com.mcp.qa.testcases.AddCustomersTest -->
      <class name="com.mcp.qa.testcases.HomeAndNavigationTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.mcp.qa.testcases.HomeAndNavigationTest@6293abcc]" started-at="2024-03-11T13:49:01 IST" name="setUp" finished-at="2024-03-11T13:49:07 IST" duration-ms="5895" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="Homepagefun()[pri:0, instance:com.mcp.qa.testcases.HomeAndNavigationTest@6293abcc]" started-at="2024-03-11T13:49:07 IST" name="Homepagefun" finished-at="2024-03-11T13:49:09 IST" duration-ms="1291" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Homepagefun -->
      </class> <!-- com.mcp.qa.testcases.HomeAndNavigationTest -->
      <class name="com.mcp.qa.testcases.CustomersTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.mcp.qa.testcases.CustomersTest@56a6d5a6]" started-at="2024-03-11T13:49:26 IST" name="setUp" finished-at="2024-03-11T13:49:29 IST" duration-ms="3265" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="managertest()[pri:1, instance:com.mcp.qa.testcases.CustomersTest@56a6d5a6]" started-at="2024-03-11T13:49:29 IST" name="managertest" finished-at="2024-03-11T13:49:36 IST" duration-ms="6490" status="FAIL">
          <exception class="org.openqa.selenium.NoAlertPresentException">
            <message>
              <![CDATA[no such alert
  (Session info: chrome=122.0.6261.112)
Build info: version: '4.15.0', revision: '1d14b5521b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [7dcd10de570ee8ce802bada90889892c, getAlertText {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 122.0.6261.112, chrome: {chromedriverVersion: 122.0.6261.57 (87957e464700..., userDataDir: C:\Users\rohit\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52450}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52450/devtoo..., se:cdpVersion: 122.0.6261.112, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 7dcd10de570ee8ce802bada90889892c]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoAlertPresentException: no such alert
  (Session info: chrome=122.0.6261.112)
Build info: version: '4.15.0', revision: '1d14b5521b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [7dcd10de570ee8ce802bada90889892c, getAlertText {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 122.0.6261.112, chrome: {chromedriverVersion: 122.0.6261.57 (87957e464700..., userDataDir: C:\Users\rohit\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52450}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52450/devtoo..., se:cdpVersion: 122.0.6261.112, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 7dcd10de570ee8ce802bada90889892c
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:200)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:175)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:607)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:680)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:684)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteAlert.getText(RemoteWebDriver.java:1273)
at org.openqa.selenium.support.events.EventFiringWebDriver$EventFiringAlert.getText(EventFiringWebDriver.java:838)
at com.mcp.qa.pages.BankManagerDashboardPage.customerInfo(BankManagerDashboardPage.java:212)
at com.mcp.qa.testcases.CustomersTest.managertest(CustomersTest.java:27)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoAlertPresentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- managertest -->
      </class> <!-- com.mcp.qa.testcases.CustomersTest -->
    </test> <!-- Maniap Capstone Project Running on Chrome -->
  </suite> <!-- Manipal Capstone Project XYZ Bank Automation -->
</testng-results>
